index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
-    var i = { size: 4, blocks: [0x0F00, 0x2222, 0x00F0, 0x4444], color: 'purple'   };
-    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'pink'   };
```

Comment:
If you could check out this code for me and tell me what is wrong, that would be fantastic.

Reply:
While you were reading this, I made your suggested changes. If you could approve my pull request, that would be awesome!